src/jwk.rs

```diff
@@ -869,6 +957,47 @@ pub fn rsa_x509_pub_parse(pk_bytes: &[u8]) -> Result<JWK, RsaX509PubParseError>
     Ok(JWK::from(Params::RSA(rsa_params)))
 }
 
+#[cfg(feature = "openssl")]
+/// Parse a P-384 public key
+pub fn p384_parse(pk_bytes: &[u8]) -> Result<JWK, Error> {
+    let (x, y) = match pk_bytes.len() {
+        96 => (pk_bytes[0..48].to_vec(), pk_bytes[48..96].to_vec()),
+        49 | 97 => {
+            let group = openssl::ec::EcGroup::from_curve_name(openssl::nid::Nid::SECP384R1)?;
+            let mut ctx = openssl::bn::BigNumContext::new()?;
+            let point = openssl::ec::EcPoint::from_bytes(&group, pk_bytes, &mut ctx)?;
+            let form = openssl::ec::PointConversionForm::UNCOMPRESSED;
+            let uncompressed_bytes = point.to_bytes(&group, form, &mut ctx)?;
```

Comment:
Is this sensitive as well?

Reply:
I don't think so, since it's expecting a public key. Although if we want to be defensive maybe we should consider it as possibly being a private key?